tests/infer/test_elbo_mapdata.py

```diff
@@ -47,7 +50,7 @@ def add_data_point(x, y):
     analytic_log_sig_n = -0.5 * torch.log(analytic_lam_n)
     analytic_loc_n = sum_data * (lam / analytic_lam_n) +\
         loc0 * (lam0 / analytic_lam_n)
-    n_steps = 7000
+    n_steps = params[0]
```

Comment:
nit: instead of using `params[0]` can you please expand `params` to `n_steps,lr` or the like?

Reply:
i.e. directly in `mark.parametrize`

Reply:
done